modules/alloy/src/main/ts/ephox/alloy/api/ui/FloatingToolbarButton.ts

```diff
@@ -61,17 +69,22 @@ const makeSandbox = (button: AlloyComponent, spec: FloatingToolbarButtonSpec, de
   const ariaControls = AriaControls.manager();
 
   const onOpen = (sandbox: AlloyComponent, toolbar: AlloyComponent) => {
+    const skipFocus = shouldSkipFocus.get().getOr(false);
```

Comment:
Is seems this variable can be computed outside of the `onOpen` callback. You can move this line out of the callback and use it inside `onOpen` and `onClose` callback as well

Reply:
Unfortunately, this won't work. Since this variable can be changed *just* before the `onOpen` and `onClose` callbacks are called. It will be always the same if we will move this line out and this is not the intention.